websites/V/VtuberWiki/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "547923574833545226",
+		"name": "char32vt"
+	},
+	"service": "VtuberWiki",
+	"description": {
+		"en": "Search or browse in depth information about Vtubers or software related to Vtubers"
+	},
+	"url": "wiki.hylia.dev",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/vr2qTO9.png",
```

Comment:
Logos should be [512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).